src/lib/httpFetcher.ts

```
@@ -0,0 +1,37 @@
+import { Fetcher, FetchResponse /*, RateLimiter */ } from './types.ts'
+
+export class HttpFetcher implements Fetcher {
+  // private rateLimiter: RateLimiter
+
+  constructor(
+    // rateLimiter: RateLimiter
+  ) {
+    // this.rateLimiter = rateLimiter
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)